File: Python/db_connection.py
import mysql.connector
from mysql.connector import Error
import pandas as pd
import requests



def get_connection():
    try:
        conn = mysql.connector.connect(
            host="localhost",
            user="root",
            password="",
            database="prediction_modeldb"
        )
        print(conn)
        return conn

    except Error as e:
        print("Connection failed:", e)
        return None

File: DBConnection.java
package predictions;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class DBConnection {

    private static final String URL = "jdbc:mysql://localhost:3306/prediction_modeldb";
    private static final String USER = "root";
    private static final String PASSWORD = "Golfaddict13!";

    public static Connection getConnection() throws SQLException {
        return DriverManager.getConnection(URL, USER, PASSWORD);
    }
}
